crates/forge/tests/e2e/common/runner.rs

```diff
@@ -31,7 +31,8 @@ pub(crate) fn test_runner(temp_dir: &TempDir) -> SnapboxCommand {
     runner(temp_dir).arg("test")
 }
 
-pub(crate) static BASE_FILE_PATTERNS: &[&str] = &["**/*.cairo", "**/*.toml"];
+pub(crate) static BASE_FILE_PATTERNS: &[&str] =
+    &["**/*.cairo", "**/*.toml", "**/data/*.json", "**/data/*.txt"];
```

Comment:
note: Needed to embed txt and json files in the docs (so they will be also copied during package setup).